### ISSUE TITLE
v5-beta.11 Unable to subscribe to network.onchange/onConnect/onDisconnect

### ISSUE THREAD

**Issue by mburger81** [danielsogl/awesome-cordova-plugins#2558]
**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://ionicworldwide.herokuapp.com/

**Current behavior:**
Using `v5-beta.11` and `ionic-angular@3.9.2` we are not able to subscribe to events for Network Plugin
```
this.network.onchange().subscribe(() => { console.log('change'); });
this.network.onConnect().subscribe(() => { console.log('onConnect'); });
this.network.onDisconnect().subscribe(() => { console.log('onDisconnect'); });
```
This are running always into error
```
TypeError: Object(...) is not a function
    at wrapEventObservable (http://localhost:8100/build/vendor.js:46601:67)
    at http://localhost:8100/build/vendor.js:46791:20
    at cordova (http://localhost:8100/build/vendor.js:146630:105)
    at Network.onDisconnect (http://localhost:8100/build/vendor.js:70177:126)
```

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Ionic:
   ionic (Ionic CLI)  : 4.0.0-rc.9 (/usr/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.10

Cordova:
   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.1.0

System:
   Android SDK Tools : 26.1.1
   NodeJS            : v8.11.1 (/usr/bin/node)
   npm               : 6.1.0
   OS                : Linux 4.15

Environment:
   ANDROID_HOME : /home/michi/software/android/sdk
```

**Comment by mburger81** [comment]
Same error running app on device with `ionic cordova run android`

**Comment by mburger81** [comment]
Also check if there is the error on **ionic4**. But the problem isn't present there, events subscription does work well.

```
Ionic:
   ionic (Ionic CLI)          : 4.0.0-rc.9 (/usr/lib/node_modules/ionic)
   Ionic Framework            : @ionic/angular 4.0.0-alpha.7
   @angular-devkit/core       : 0.7.0-rc.0
   @angular-devkit/schematics : 0.7.0-rc.0
   @angular/cli               : 6.0.8
   @ionic/ng-toolkit          : 1.0.0-rc.8
   @ionic/schematics-angular  : 1.0.0-rc.8

Cordova:
   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0

System:
   Android SDK Tools : 26.1.1
   NodeJS            : v8.11.1 (/usr/bin/node)
   npm               : 6.1.0
   OS                : Linux 4.15

Environment:
   ANDROID_HOME : /abc/software/android/sdk

```

**Comment by meadleyson** [comment]
Confirming this issue is still present

**Comment by jmesa-sistel** [comment]
@danielsogl 
The bug is this commit:
[chore(common): fix linter error (#2941)](https://github.com/ionic-team/ionic-native/commit/e1a25af0b5b3166517fec993dceea075ff26627b)
Now element is a boolean (usually true) always.
If you revert it it is fixed.
